Fixed typo (#476)
bigcommerce/dev-docs: docs/stencil-docs/storefront-customization/custom-templates.md

## Authoring a Custom Template

> As a theme developer, you must first create the custom subdirectory in the templates/pages directory, and four required subdirectories inside of it (brand, category, product, page), which will result in the following directory paths:

Reviewer: ```suggestion
As a theme developer, you must first create the custom subdirectory in the `templates/pages` directory, and the four required subdirectories inside of it (brand, category, product, page). Creating these subdirectories will result in the following directory paths:

```

* After editing your .stencil file, you must restart stencil to see your changes locally. Enter stencil start on the command line, appending any appropriate switches for your workflow (e.g.: stencil start -e -n).

### Why These URL Requirements?

> When you create a local custom template page for products, you expect that page to have access to all Stencil product objects. However, the server cannot readily determine the page type of each local custom template. So we give it a hint. We instruct the server to look at the page type of the URL to which you have mapped the template.

Reviewer: ```suggestion
When you create a local custom template page for products, you expect that page to have access to all Stencil product objects. However, the server cannot readily determine the page type of each local custom template. So we give it a hint by instructing the server to look at the page type of the URL where you mapped the template.

```
